public/apps/account/account-nav-button.tsx

```diff
@@ -186,3 +185,24 @@ export function AccountNavButton(props: {
     </EuiHeaderSectionItemButton>
   );
 }
+
+export function reloadAfterTenantSwitch(): void {
```

Comment:
where will this function be called? I see that you explicitly call it in tests, but I was not sure where will be called